README.markdown

```diff
@@ -173,11 +173,14 @@ This documentation assumes there's a `yokozuna` bucket type created and activate
 ``` ruby
 # Create a client and bucket.
 client = Riak::Client.new
-bucket = client.bucket 'pizzas'
+bucket_type = client.bucket_type 'yokozuna'
+bucket = bucket_type.bucket 'pizzas'
 
 # Create an index and add it to a typed bucket. Setting the index on the bucket
 # may fail until the index creation has propagated.
-client.create_search_index 'pizzas'
+index = Riak::Search::Index.new 'pizzas'
+index.exist? #=> false
+index.create!
 client.set_bucket_props bucket, {search_index: 'pizzas'}, 'yokozuna'
```

Comment:
I agree with Sean regarding the extra yokozuna argument, it feels "janky" and prone to misuse / confusion. Could the bucket not be aware of its type? e.g. maybe the type object is a private member of the bucket?